### PR TITLE
[rfr] Minor improvements

### PR DESCRIPTION
**Pull request by haphan**

### CODE REVIEW

**Comment by haphan** [comment]
Please review @jamiehannaford

File: src/BlockStorage/v2/Api.php [hunk 1]
```diff
@@ -50,19 +50,23 @@ public function getVolumesDetail(): array
...
         return [
-            'method' => 'GET',
-            'path'   => 'volumes/{id}',
-            'params' => ['id' => $this->params->idPath()],
+            'method'     => 'GET',
+            'path'       => 'volumes/{id}',
+            'params'     => [
+                'id' => $this->params->idPath(),
+            ],
+            'allTenants' => $this->params->allTenants(),
```
Comment (jamiehannaford) [inline src/BlockStorage/v2/Api.php]: shouldn't this be in the params array?

File: src/Common/Api/AbstractParams.php [hunk 2]
```diff
@@ -97,4 +97,14 @@ public function sortKey(): array
...
         ];
     }
+
+    public function allTenants(): array
+    {
+        return [
+            'type'        => self::STRING_TYPE,
+            'location'    => self::QUERY,
+            'sentAs'      => 'all_tenants',
+            'description' => '(Admin only) Set this to 1 to pull volumes information from all tenants.',
```
Comment (jamiehannaford) [inline src/Common/Api/AbstractParams.php]: s/volumes/volume/

File: src/Common/Api/AbstractParams.php [hunk 3]
```diff
@@ -97,4 +97,14 @@ public function sortKey(): array
             'description' => "Sorts by one or more sets of attribute and sort direction combinations.",
         ];
     }
+
+    public function allTenants(): array
+    {
+        return [
+            'type'        => self::STRING_TYPE,
```
Comment (jamiehannaford) [inline src/Common/Api/AbstractParams.php]: if we use a bool, won't it get converted to either 1 or 0 anyway?

**Comment by haphan** [comment]
@jamiehannaford comments updated.

**Pull request merged**